db/migrate/20220811080557_add_validated_at_to_planning_application.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddValidatedAtToPlanningApplication < ActiveRecord::Migration[6.1]
+  def up
+    add_column :planning_applications, :validated_at, :datetime
+
+    PlanningApplication.all.find_each do |p|
+      p.update!(validated_at: p.documents_validated_at&.to_datetime)
+    end
+  end
```

Comment:
Is the plan to remove `documents_validated_at` in a separate change once we know this has worked?

Reply:
Doh, sorry, have just read the description and seen that you already answered this!

Reply:
yes I will do it on a separate PR as I want to have it on a different transaction to avoid downtime in migrations 😄